.github/workflows/rust.yml

```diff
@@ -39,6 +39,7 @@ jobs:
         uses: actions-rs/cargo@v1
         with:
           command: check
+          args: --tests
```

Comment:
What do you think about going full-monty here and do `--all-targets`? Docs say: "Check all targets. This is equivalent to specifying --lib --bins --tests --benches --examples."

Reply:
works for me!